ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/proxy/security-token-service/cucushift-installer-rehearse-aws-ipi-proxy-security-token-service-workflow.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/rehearse/aws/ipi/proxy/security-token-service/cucushift-installer-rehearse-aws-ipi-proxy-security-token-service-workflow.yaml",
```

Comment:
In this PR, using `security-token-service` naming, compared with https://github.com/openshift/release/pull/27914/files, it was using `cco-manual-workload-identity`, I think we need to create a consistent name convention. How about like this:
```
gcp/ipi/cco-manual/security-token-service/
aws/ipi/proxy/cco-manual/security-token-service/
```
cc @jianli-wei to be noticed.

Reply:
@jianlinliu FYI The words "workflow identity" were from [OCP doc](https://docs.openshift.com/container-platform/4.10/authentication/managing_cloud_provider_credentials/cco-mode-gcp-workload-identity.html).

Reply:
yes, for different providers, it has different naming convetion, for aws, it called security token service, for gcp, it called workload identity, if we can refer dev naming convetion, using oidc-creds(OpenID Connect) instead?

Reply:
Good to know. I mentioned here, just for easy to mapping our QE's test plan, if I remember in our QE's ci profiles, we always named them as `STS` whatever on which cloud providers. 

I also agree that our workflow name should be keep consistent with official doc as much as possible.

So here how about rename it to `ipi/proxy/manual-cco-security-token-service` on aws, so that users get know it is dedicated for manual cco testing.

Reply:
sure, that makes sense, i will change it to `ipi/proxy/manual-cco-security-token-service` for aws, so that also keep consisitent with gcp